ManatiCR/artavia
# AR-73 listado publicaciones

[AR-73](https://manati.atlassian.net/browse/AR-73)
[Figma designs](https://www.figma.com/file/sRoXqR6hyiNvc8moaFeZA6/Artavia-Consultor%C3%ADa)
-
  This PR depends of this one: [AR-16](https://github.com/ManatiCR/artavia/pull/6)
---
**What was done**
- Create the contet type publicaciones
- Create the list of publicaciones, publicaciones page.
- Page to test: http://artavia.docker/es/publicaciones and the display node of publicaciones

**Steps to test**
- Run `composer update && ./scripts/artavia_local_install.sh` new module was added 
if composer update does't work run `sudo rm -r web && rm composer.lock && composer install && ./scripts/artavia_local_install.sh`
- Create publicaciones nodes (You can use devel) but remember add at least two nodes with a title you can remember and add image to the body field (At least two) as the following way:
![screenshot from 2018-01-11 17-03-24](https://user-images.githubusercontent.com/9342250/34851821-621392a8-f6f1-11e7-9579-3e57b589bd37.png)
- Each node display of publicaciones should show the followings fileds:
  - Título*- Text field 
  - Fecha de la publicación* - calendario (predeterminada la fecha actual)
  -  Autor* - Referencia a término (vocabulario abierto "Autor")
  - Tipo de Publicación* (dropdown: artículo, noticia, opinión, análisis, otro (agregar otro))
  - Resumen: tres campos de texto, uno requerido mínimo
  - Editor de texto*: permite subtítulo, cuerpo e imágenes
  - Tags*: máximo de 5 tags
- After that go to: http://artavia.docker/es/admin/structure/entityqueue/publicaciones_destacadas/publicaciones_destacadas and add the two publicaciones which names you remember to that entityque
- Now go to http://artavia.docker/es/publicaciones and check the publicaciones you add to the entityque are being displayed first and showing the first image of the body.
- The other publicaciones should show only the text of the body not any image.
- The publiciones destacadas should be displayed only in the first page (if you are using the pager)
- Go to any publicacion node and click in one of the tags, you should go to the publicaciones list page with a filter applied by the corresponding taxonomy term
